spaceR/spaceR/Controllers/LoginViewController.swift

```diff
@@ -7,11 +7,18 @@
 
 import UIKit
 
-class LoginViewController: UIViewController, UITextFieldDelegate {
+class LoginViewController: UIViewController, UITextFieldDelegate, ViewModelDelegate {
```

Comment:
Rather conform to these protocols in an extension underneath this class: 

`extension LoginViewController: UITextFieldDelegate { }`
`extension LoginViewController: ViewModelDelegate { } `